resources/upgrading-to-aep/manual-lifecycle-implementation.md

```diff
@@ -0,0 +1,280 @@
+# Manual Lifecycle Implementation in the Experience Platform SDKs
+
+In the Experience Platform SDK, all Lifecycle implementation must be done manually. This page provides a suggested implementation of Lifecycle. The correct implementation for Lifecycle should ultimately be determined by the app developer.
+
+{% hint style="danger" %}
+In version 4 of the iOS SDK, this implementation was done for your automatically.
+
+When upgrading to the Experience Platform SDK, you must add code to continue collecting Lifecycle metrics.
+{% endhint %}
+
+## Importing and Registering the Lifecycle Extension
+
+{% tabs %}
+
+{% tab title="Android" %}
+#### Java
+
+Import the Lifecycle framework:
+
+```java
+import com.adobe.marketing.mobile.*;
+```
+
+Register the framework with Mobile Core:
+
+```java
+public class MyApp extends Application {​
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        MobileCore.setApplication(this);
+
+        try {
+            Lifecycle.registerExtension();
+        } catch (Exception e) {
+            // Log the exception
+        }
+    }
+}
+```
+
+{% endtab %}
+
+{% tab title="iOS (Objective-C)" %}
+#### Objective-C
+
+Import the Lifecycle framework:
+
+```objectivec
+#import "ACPLifecycle.h"
+```
+
+Register the Lifecycle extension with the SDK Core by adding the following to your app's `application:didFinishLaunchingWithOptions:` delegate method:
+
+```objectivec
+- (BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    // register the lifecycle extension
+    [ACPLifecycle registerExtension];
+}
+```
+{% endtab %}
+
+{% tab title="iOS (Swift)" %}
+#### Swift
+
+In Swift, importing `ACPCore` also imports the necessary Lifecycle APIs:
+
+```swift
+import ACPCore
+```
+
+Register the Lifecycle extension with the SDK Core by adding the following in your app's `application:didFinishLaunchingWithOptions:` delegate method:
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    // register the lifecycle extension
+    ACPLifecycle.registerExtension();
+}
+```
+{% endtab %}
+
+{% tab title="React Native" %}
+#### JavaScript
+
+Import the Lifecycle extension:
+
+```jsx
+import {ACPLifecycle} from '@adobe/react-native-acpcore';
+```
+
+Register the extension with Core:
+
+```jsx
+ACPLifecycle.registerExtension();
+```
+{% endtab %}
+
+{% endtabs %}
+
+## Start Lifecycle Collection
+
+You can start collecting Lifecycle information at any time in your app, but it is recommended to start as soon as the app enters the foreground. This allows the lifecycle metrics to be properly attributed to all of the user's activity for their current session.
```

Comment:
Please reword as follows:

"You can start collecting Lifecycle information at any time in your app, but we recommended that you start as soon as your app enters the foreground. This allows lifecycle metrics to be correctly attributed to all of your users' activities for their current session."